README.md

```diff
@@ -128,13 +130,39 @@ also include a binary payload denoting whether or not the expectation
 passed or failed.
 
 ```rust
-try_expect!(
+expect!(
     probe,
     PRODUCER_TX_STATUS_OK,
     tx_status.is_ok(),
     "Measurement producer send result status",
     tags!("producer", "SEVERITY_10")
-)?;
+);
+```
+
+### Recording Wall Clock Time
+
+Wall clock time can be recorded as a standalone timestamp or
```

Comment:
Could we add a few words about recommended best practices for clock behavior? E.G. "Please make these numbers go up, and not down if you're faking it." E.G. "Use a consistent clock-source for all the time-related measurements at a given probe (don't mix and match clock-sources for a probe).